src/settings/toml/tests/deployments.rs

```diff
@@ -13,18 +13,18 @@ const ACCOUNT_ID: &str = "fakeaccountid";
 
 // TOP LEVEL TESTS
 #[test]
-fn it_errors_on_empty_deploy_config() {
+fn it_errors_on_empty_get_deployments() {
     let test_toml = WranglerToml::webpack("empty");
     let toml_string = toml::to_string(&test_toml).unwrap();
     let manifest = Manifest::from_str(&toml_string).unwrap();
 
     let environment = None;
 
-    assert!(manifest.deploy_config(environment).is_err());
+    assert!(manifest.get_deployments(environment).is_err());
 }
 
 #[test]
-fn it_errors_on_conflicting_deploy_configs() {
+fn it_errors_on_conflicting_get_deploymentss() {
```

Comment:
update the name of the test to reflect expected behavior (we no longer expect this scenario to error, but rather to allow many kinds of deploy target).